app/routes/events/view/speakers/list.js

```diff
@@ -11,39 +11,55 @@ export default Route.extend({
         return this.get('l10n').t('Rejected');
     }
   },
-  model(params) {
+  async model(params) {
```

Comment:
Why is `async` required here?

Reply:
@mayank8318 I thought to filter on the basis of a related model async might be required but it's not so. Please review now. :sweat_smile: